### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by stutisk**
image
Added Sidebar , Navbar Component and hero section
Will be adding Categories section

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Good work with UI
Very Neat n Clean Code
Good to merge

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+              <div class="input-group relative flex items-stretch w-full mb-4">
+                <input
+                  type="search"
+                  class=" rounded-l-full form-control relative flex-auto min-w-0 block w-full px-3 py-1.5 text-base font-normal text-slate-400 bg-white bg-clip-padding border border-solid border-gray-300  transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+                  placeholder="Search"
+                  aria-label="Search"
+                  aria-describedby="button-addon2"
+                />
+                <button
+                  class="btn rounded-r-full inline-block px-6 py-2.5 bg-sky-500 text-white font-medium text-xs leading-tight uppercase  shadow-md  focus:bg-sky-500 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-sky-600 active:shadow-lg transition duration-150 ease-in-out flex items-center"
```
Comment (ashishnnnnn) [inline src/components/Navbar/Navbar.jsx]: this class attribute should be changed to className in JSX.

File: src/Pages/Homepage/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                  <h1 className="text-5xl text-slate-800 ">
+                    Welcome To Galaxia{" "}
+                    <span className="text-sky-500">gana</span>
+                  </h1>
+                  <p className="py-4 font-medium text-2xl text-slate-800">Life is Better With Music </p>
+                 
+                  <button
+                    type="button"
+                    class="  inline-block px-6 py-2.5 border-2 border-slate-900  text-slate-800 font-medium text-xl   rounded hover:bg-black  hover:text-slate-50 hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out"
+                  >
```
Comment (ashishnnnnn) [inline src/Pages/Homepage/Homepage.jsx:34]: Nice use of tailwind classes...

File: src/components/Sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            <span className="group-hover:bg-sky-500 rounded-lg p-1">
+              <BiLike size={26} />
+            </span>{" "}
+            Liked Videos
+          </li>
+          <li className="py-4 flex flex-row gap-4 cursor-pointer group hover:text-slate-50 ">
+            {" "}
+            <span className="group-hover:bg-sky-500 rounded-lg p-1">
+              <MdWatchLater size={26} />{" "}
+            </span>{" "}
```
Comment (ashishnnnnn) [inline src/components/Sidebar/Sidebar.jsx:46]: Nice hack to introduce space..

**Review by ashishnnnnn — APPROVED** [review]
Code seems pretty much clean..
Good Use Of Tailwind..

**Review by lomesshh — APPROVED** [review]
clean and modular code
UI looks good
looks good to merge

**Pull request merged**